### ISSUE TITLE
Question about dataset

### ISSUE THREAD

**Issue by Rayoll** [swuxyj/DeepHash-pytorch#24]
If I have an image classification dataset, how could I get the onehot label of each image for retrieval?

**Comment by swuxyj** [comment]
I hope [code1](https://github.com/swuxyj/DeepHash-pytorch/blob/master/data/mirflickr/code.py) 、 [code2](https://github.com/swuxyj/DeepHash-pytorch/blob/master/data/voc2012/code.py) 、 [code3](https://github.com/swuxyj/DeepHash-pytorch/blob/master/data/nuswide_21/code.py)  can help you.

**Comment by uniartisan** [comment]
> I hope [code1](https://github.com/swuxyj/DeepHash-pytorch/blob/master/data/mirflickr/code.py) 、 [code2](https://github.com/swuxyj/DeepHash-pytorch/blob/master/data/voc2012/code.py) 、 [code3](https://github.com/swuxyj/DeepHash-pytorch/blob/master/data/nuswide_21/code.py) can help you.

Thanks a lot, it really help.
Could you please tell me what does database.txt do in training the network :)

**Comment by swuxyj** [comment]
> > I hope [code1](https://github.com/swuxyj/DeepHash-pytorch/blob/master/data/mirflickr/code.py) 、 [code2](https://github.com/swuxyj/DeepHash-pytorch/blob/master/data/voc2012/code.py) 、 [code3](https://github.com/swuxyj/DeepHash-pytorch/blob/master/data/nuswide_21/code.py) can help you.
> 
> Thanks a lot, it really help. Could you please tell me what does database.txt do in training the network :)

训练的时候只用了：train_loader
test_loader, dataset_loader 在算mAP的时候用的：
```python
            tst_binary, tst_label = compute_result(test_loader, net, device=device)

            # print("calculating dataset binary code.......")\
            trn_binary, trn_label = compute_result(dataset_loader, net, device=device)

            # print("calculating map.......")
            mAP = CalcTopMap(trn_binary.numpy(), tst_binary.numpy(), trn_label.numpy(), tst_label.numpy(),
                             config["topK"])
```

**Comment by BigBoomDream** [comment]
hi, thank you very much for your work, the database.txt test.txt and train.txt of cifar10 in data is not given

**Comment by swuxyj** [comment]
> hi, thank you very much for your work, the database.txt test.txt and train.txt of cifar10 in data is not given  

You can get the cifar10 dataset by ```import torchvision.datasets```, and then directly divide it in multiple ways. You can refer to the [code](https://github.com/swuxyj/DeepHash-pytorch/blob/master/utils/tools.py#L114), of course, You can also generate these files yourself.
